build.sbt

```diff
@@ -24,7 +24,7 @@ lazy val nettyTcnativeVersion = "2.0.29.Final"
 
 lazy val commonSettings = Seq(
   organization := "com.google.cloud.spark",
-  version := "0.17.4-SNAPSHOT",
+  version := "0.17.5-SNAPSHOT",
```

Comment:
Please remove, the next version is either 0.17.4 or 0.18.0

Reply:
Done.